### ISSUE TITLE
Combine-trailer relationship:

### ISSUE THREAD

**Issue by huseyinsayar** [Courseplay/courseplay#1367]
Combine-trailer relationship:
Can the tractor, which is going to empty the harvester tank, prevent the cropping of the cultivated fields in the field?

Biçerdöver-Romörk ilişkisi:
Biçerdöver tankını boşlatmaya giden traktörün, tarlada ki ekili alanları ezmesi engelenemez mi?

**Comment by Satissis** [comment]
Are you talking about the crop destruction when a vehicle drives over an field ?

**Comment by SquareCloud506** [comment]
The pathfinding option in CP for FS15 already does this. It prevents the unloading tipper from approaching the combine to unload if the unloading pipe is facing a crop waiting to be harvested. It only didn't work under 2 conditions. 1, when the harvester hit 100% before reaching the end of the row and still having crops on the unloading side, and 2, when a harvester hit 50% before reaching the end of the row, but made the turn for the next row before the tipper got to the harvester (this would result in the tipper making the turn and following to unload in the crops). Using pathfinding, with a well placed entry point for the tipper to the path of the combine would keep the tipper out of the crops most of the time.

Also, crop destruction is a mod, and can be disabled if it's a real problem when using CP.

If I were to suggest any changes to pathfinding, I would suggest, 1) have the harvester wait at the end of the row if a tipper has been called but hasn't reached it yet, thus avoiding the tipper turning the corner and following in the crops, and 2) if the harvester reaches 100% and having the tipper enter the crops to unload, then have the harvester stay until is empty or tipper leaves for being full, to keep the tipper from following too much in crops. 

Of course, I enjoyed CP for FS15 so much that I learned to live with the little mistakes it would make. Patiently waiting for a chance to try CP with FS17. 

Cheers.
